R/tellTS.R

```diff
@@ -21,8 +21,8 @@ tellTS.default <- function(x, ts.matrix, fun,
   ))
   switch(parallel,
     "clusterApply" = {
-      clusterEvalQ(cl, library(sensitivity))
-      clusterEvalQ(cl, library(ff))
+      clusterEvalQ(cl, requireNamespace(sensitivity))
+      clusterEvalQ(cl, requireNamespace(ff))
       clusterExport(cl, c("ts.matrix", "x", "fun"), envir = environment())
```

Comment:
This is a tricky one. These are actually commands to be run on a remote server, so we probably need some kind of additional error handling if requireNamespace is used instead of library.

Reply:
Yes, I wasn't sure here. if you use foreach you can simply export the libraries within the call, is there a way we can rewrite the cluster calls as a foreach?
Otherwise we need to work out a different way to export the libraries to the nodes

Reply:
Second thought, do we need to replace library() here? Maybe this would not create an error in R CMD check?

Reply:
As discussed, please do see what R CMD check says.
The right solution does seem to be to check the output of the clusterEvalQ call and throw an error if needed.

library(sensitivity) could be removed by using sensitivity::tell but library(ff) is needed in order for tell to be able to read the ts.matrix I believe, i.e. there's no explicit ff calls.

Feel free to just not fix this one and we can accept the rest of the changes.

Reply:
Still to run R CMD CHK, fails at the moment on other issues, does not flag the "library" issue